draft-ietf-quic-multipath.md

```diff
@@ -127,11 +127,36 @@ path management in {{Section 9 of QUIC-TRANSPORT}} and therefore
 requires negotiation between the two endpoints using a new transport
 parameter, as specified in {{nego}}.
 
-This extension uses multiple packet number spaces.
-When multipath is negotiated, each separate packet number space is linked to a path ID.
+This extension specifies a new Path Identifier (Path ID), which is an
+integer between 0 and 2^32 - 1 (inclusive). Path identifies are generated
+monotonically increasing and cannot be reused.
+
+The Path ID is used to
+address a path in the new multipath control frames,
+such as PATH_ABANDON {{path-abandon-frame}}, PATH_STANDBY {{path-standby-frame}}},
+PATH_AVAILABLE {{path-available-frame}} as well as ACK_MP {{ack-mp-frame}}.
+Further, Connection IDs are issued per Path ID.
```

Comment:
We may discuss that later, but connection IDs are issued for the connection, because they must be unique withing the scope of the connection, and attached to a path-id.